secretservice/secretservice.go

```diff
@@ -33,8 +33,10 @@ func (h Secretservice) Add(creds *credentials.Credentials) error {
 	defer C.free(unsafe.Pointer(username))
 	secret := C.CString(creds.Secret)
 	defer C.free(unsafe.Pointer(secret))
+	displayLabel := C.CString("Docker credentials for " + creds.ServerURL)
```

Comment:
looking now at it; should it be either configurable, or more generic? (registry credentials for)?

Reply:
Or alternatively, we could use the name of the binary perhaps (`docker-credentials-secretservice` by default)


Just thinking out loud here 🤗

Reply:
Hum yeah "Registry credentials" sounds better indeed

Reply:
Updated 👍